### PR TITLE
docs(guides): add guides for getting started

### PR DESCRIPTION
**Pull request by jeffbcross**

### CODE REVIEW

**Comment by jeffbcross** [comment]
Still needs Service Worker

File: guides/app-shell.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+Now let's add the toolbar to our component.
+Open src/app/hello-mobile.component.ts in your editor.
+We first need to import the directives from the toolbar package like so:
+
+```typescript
+import {Component} from 'angular2/core';
+import {RouteConfig, ROUTER_DIRECTIVES, ROUTER_PROVIDERS} from 'angular2/router';
+import {MdToolbar} from '@angular2-material/toolbar';
+
```
Comment (gkalpak) [inline guides/app-shell.md]: Missing closing `` ``` ``

File: guides/app-shell.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+And then we need to register the directive with our component so it can be available in the component's view. While we're at it we'll delete the moduleId, templateUrl and styleUrls properties to instead use inline template and styles. Here is what the Component decorator should look like before we add the template:
+
+@Component({
```
Comment (gkalpak) [inline guides/app-shell.md]: Missing opening \`\`\`.

File: guides/app-shell.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    <md-toolbar>
+      App Shell Demo
+    </md-toolbar>
+  `,
+  directives: [MdToolbar]
+})
+```
+
+Now serve the app again with ng serve, and you should see something like this:
+
```
Comment (gkalpak) [inline guides/app-shell.md]: Like what ?

File: guides/app-shell.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+  packages: Object.assign({
+    '@angular2-material/toolbar': {
+      defaultExtension: 'js',
+      main: 'toolbar.js'
+    }
+  }, createPackageConfig(barrels))
+};
+```
+
+This tells the SystemJS JavaScript loader how to load assets when a module imports ‘@angular2-material/toolbar'.
```
Comment (mgechev) [inline guides/app-shell.md]: Different quote symbols used: '‘" and "'".

File: guides/app-shell.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+barrelConfig[barrelName] = {
+      format: 'cjs',
+      defaultExtension: 'js',
+      main: 'index'
+    };
+```
+
+Now let's add the plugin to our build pipeline. Open angular-cli-build.js in your editor. This file is the CLI's entry point to modify the build process. By default, the module just exports an instance of Angular CLI's default Angular2App class, which is a broccoli tree. We want to modify our process to return a merged tree, which will overwrite the index.html of the Angular2App tree with the app shell tree.  First, we'll import the 2 broccoli plugins we just installed:
+
+```typescript
```
Comment (mgechev) [inline guides/app-shell.md]: The block isn't closed with `\`\`\``.

File: guides/app-shell.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+var Angular2App = require('angular-cli/lib/broccoli/angular2-app');
+var AppShellPlugin = require('angular2-broccoli-prerender').AppShellPlugin;
+var mergeTrees = require('broccoli-merge-trees');
+
+Now let's use the AppShellPlugin to create a new tree with our completely-inlined index.html.
+
+module.exports = function(defaults) {
```
Comment (mgechev) [inline guides/app-shell.md]: The block isn't opened with ```typescript.

File: guides/app-shell.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+  });
+  var appShell = new AppShellPlugin(ngTree, 'index.html', 'main-app-shell');
+  return mergeTrees([ngTree, appShell], {
+    overwrite: true
+  });
+};
+```
+
+
+Now if we run ng serve, and browse to localhost:4200, we should still see the basic component with a toolbar. But when we open dist/index.html, we should see our app component and all styles for the toolbar completely inlined in the document.
```
Comment (mgechev) [inline guides/app-shell.md]: `localhost:4200` can be a link?

**Pull request closed without merging**